vividus/src/main/java/org/vividus/bdd/groovy/VariableContextAwareGroovyScriptEvaluator.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2019-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.bdd.groovy;
+
+import org.springframework.scripting.ScriptSource;
+import org.springframework.scripting.groovy.GroovyScriptEvaluator;
+import org.vividus.bdd.context.IBddVariableContext;
+
+public class VariableContextAwareGroovyScriptEvaluator extends GroovyScriptEvaluator
```

Comment:
is `GroovyScriptEvaluator` thread-safe? (in other words: is it safe to use it for multiple evaluations and from different threads?)

Reply:
![image](https://user-images.githubusercontent.com/17816729/97004136-3f11f400-1545-11eb-93b7-682e16ac2879.png)
![image](https://user-images.githubusercontent.com/17816729/97004153-4802c580-1545-11eb-8a61-a27cc431fde2.png)

Reply:
100 threads